subprojects/core/src/main/java/org/gradle/internal/classpath/DefaultClasspathTransformerCacheFactory.java

```diff
@@ -44,6 +44,7 @@
         .incrementedIn("3.2-rc-1")
         .incrementedIn("3.5-rc-1")
         .changedTo(8, "6.5-rc-1")
+        .changedTo(10, "6.8-rc-1")
```

Comment:
Rather than increment this version when the instrumentation changes, we've been changing what is written to the hasher in `InstrumentingClasspathFileTransformer` when it calculates its config hash.

Reply:
Thanks, fixed!